testing/bsg_noc/bsg_wormhole_router_adapter_in/Makefile

```diff
@@ -1,4 +1,9 @@
+export BSG_IP_CORES_DIR=/home/petrisko/bitbucket/pre-alpha-release/basejump_stl
```

Comment:
I don't think we want these 2 paths in basejump.
Also in the adapter_out testbench

Reply:
Good catch :)